scripts/changeset/get-full-changelog.mts

```diff
@@ -0,0 +1,154 @@
+  const isBreaking = title.includes(`${prType}!`);
+
+  const titleDescription = title.replace(/\w+\W+(.*)/, '$1'); // chore!: add something -> add something
+  const summary = titleDescription.charAt(0).toUpperCase() + titleDescription.slice(1);
+
+  const markdown = `- ${prLink}, ${summary}, by ${user}`;
+  return {
+    prType,
+    isBreaking,
+    markdown,
+  };
+}
+
+function sortChangelogsByPrType(a: ChangelogInfo, b: ChangelogInfo) {
+  const aIdx = prTypes.indexOf(a.prType);
+  const bIdx = prTypes.indexOf(b.prType);
+
+  if (aIdx === -1) {
+    return bIdx === -1 ? 0 : 1;
+  }
+
+  if (bIdx === -1) {
+    return -1;
+  }
+
+  return aIdx - bIdx;
+}
+
+async function getChangelogs(octokit: Octokit, changesets: NewChangeset[]) {
+  const changesetsWithReleases = changesets.filter((x) => x.releases.length > 0);
+  const changelogs = await Promise.all(
+    changesetsWithReleases.map(async (changeset) => getChangelogInfo(octokit, changeset))
+  );
+
+  return changelogs.sort(sortChangelogsByPrType);
+}
+
+function mapPrTypeToTitle(prType: string) {
+  switch (prType) {
+    case 'feat':
+      return 'Features';
+    case 'fix':
+      return 'Fixes';
+    case 'chore':
+      return 'Chores';
+    case 'docs':
+      return 'Docs';
+    case 'ci':
+      return 'CI';
+    default:
+      return 'Misc';
+  }
+}
+
+function groupChangelogsForListing(changelogs: ChangelogInfo[]) {
+  const prTypeWithChangelogs = prTypes.reduce(
+    (acc, prType) => {
+      acc[prType] = changelogs.filter((c) => c.prType === prType).map((c) => c.markdown);
+      return acc;
+    },
+    {} as Record<string, string[]>
+  );
+  
+
+  return Object.entries(prTypeWithChangelogs)
+    .filter(([, c]) => c.length > 0)
+    .map(([prType, c]) => [mapPrTypeToTitle(prType), c] as [string, string[]]);
+}
+
+function listBreakingMd(changelogs: ChangelogInfo[]) {
+  const changelogGroups = groupChangelogsForListing(changelogs.filter((x) => x.isBreaking));
+
+  return changelogGroups
+    .map(
+      ([groupTitle, c]) => `- ${groupTitle}
+${c.map((x) => `    ${x}`).join('\n')}`
+    )
+    .join('\n')
+    .trim();
+}
+
+function listNonBreakingMd(changelogs: ChangelogInfo[]) {
+  const changelogGroups = groupChangelogsForListing(changelogs.filter((x) => !x.isBreaking));
+
+  return changelogGroups
+    .map(
+      ([groupTitle, c]) => `# ${groupTitle}
+${c.join('\n')}`
+    )
+    .join('\n\n')
+    .trim();
+}
+
+export async function getFullChangelog(octokit: Octokit) {
+  const changesets = await getChangesets(process.cwd());
+
+  const changelogs = await getChangelogs(octokit, changesets);
+
+  const breaking = listBreakingMd(changelogs);
+  const nonBreaking = listNonBreakingMd(changelogs);
+
+  let content = `# RELEASE - ${process.env.RELEASE_TAG ?? 'TBD'}\n\n`;
+  content += breaking ? `# Breaking\n\n${breaking}` : '';
+  content += breaking && nonBreaking && '\n\n---\n\n';
+  content += nonBreaking;
+
+  console.log(content)
```

Comment:
Was this an intentional `console.log`?
If it is, I'd say it should be performed on the entrypoint (`update-changelog.mts`), rather than in the method.

```suggestion

```

Reply:
https://github.com/FuelLabs/fuels-ts/pull/1872/commits/e4356646f09fc009492d839d2d8bfdacbf1a1e4c